assets/src/components/reporting/ReportingDashboard.tsx

```diff
@@ -171,6 +183,16 @@ class ReportingDashboard extends React.Component<Props, State> {
     });
   };
 
+  formatResponseTimeByWeekDay = (stats: Array<ResponseTimeByWeekDay>) => {
+    return stats.map((ele) => {
```

Comment:
not important, but one useful js/ts thing you can do here is just destructure `(ele) =>` into `({average, day}) =>` (might improve readability a bit, since `ele` isn't particularly descriptive)